alpha/apps/kaltura/lib/KExternalErrors.class.php

```diff
@@ -59,6 +59,7 @@ class KExternalErrors
 	const BUNDLE_CREATION_FAILED = 47;
 	const ENTRY_NOT_SEQUENCE = 48;
 	const INVALID_MIN_BITRATE = 49;
+	const INVALID_PARAMETER = 50;
```

Comment:
You forgot to add this value to the errorDescriptionMap as well.
You can use the [%s] to output a more meaningful error that will reflect what you expected to get.